server/container/guice/cassandra-guice/pom.xml

```diff
@@ -246,11 +246,11 @@
         </dependency>
         <dependency>
             <groupId>${james.groupId}</groupId>
-            <artifactId>james-server-task-cassandra</artifactId>
+            <artifactId>james-server-task-distributed</artifactId>
```

Comment:
This is not OK: this will bring the RabbitMQ dependency in a non-rabbitMQ project.

Please move code related to 'distributed task management tests' to `cassandra-rabbitmq-guice` which is a much more suitable location.

Reply:
(sorry for the late remark)